src/components/Form/CalendarInput/CalendarInput.js

```diff
@@ -197,23 +198,45 @@ CalendarInput.defaultProps = {
 };
 
 CalendarInput.propTypes = {
+    /** If true ,there is a "x" mark inside the component,
+     *  if "x" mark is clicked the element inside the component will be cleared,
+     *  if false there is no "x" mark in the component.*/
```

Comment:
R: Set to false to remove the clear button to disable clearing of the value through the component.

Reply:
Unresolved as of 1108 30 July.